master/usage/policy/whitelist-comms.md

```diff
@@ -0,0 +1,202 @@
+---
+title: Securing Calico's own communications
+---
+
+## About securing {{site.prodname}}'s own communications
+
+By generalizing the approach to [securing {{site.prodname}}'s metrics
+endpoints](secure-metrics), {{site.prodname}}'s own communications can be secured by
+{{site.prodname}} policy.  This defends against someone attacking the cluster by trying to
+impersonate a {{site.prodname}} component; the {{site.prodname}} policy will prevent them
+from successfully connecting to real {{site.prodname}} components, and they will not be
+able to exfiltrate privileged information, or to modify the cluster's networking or
+security.
+
+## Protected servers
+
+We use the term 'protected server' for each {{site.prodname}} service that we want to
+protect: {{site.prodname}}'s etcd servers and any other {{site.prodname}} components that
+accept incoming connections.  For example, the {{site.nodecontainer}} Prometheus metrics
+endpoints.  A protected server runs on certain hosts within the cluster, and serves on a
+particular TCP or UDP port.
+
+## Allowing other traffic by default
+
+You need to [decide how to handle traffic that is _not_ to a protected
+server](secure-metrics#choosing-an-approach).
+
+If you are [using a port blacklist
+approach](secure-metrics#using-a-port-blacklist-approach), configure a default allow
+policy like this:
+
+```yaml
+apiVersion: projectcalico.org/v3
+kind: GlobalNetworkPolicy
+metadata:
+  name: bl-default-allow
+spec:
+  selector: bl-default-allow == "true"
+  order: 5000
+  ingress:
+  - action: Allow
+  egress:
+  - action: Allow
+```
+
+We will add a `bl-default-allow == "true"` label to each HostEndpoint where traffic should
+be allowed by default.
+
+## Policy for each protected server
+
+For each protected server `xyz`, configure a policy that only allows access to that server
+from clients with the label `allowed-xyz-client`, like this (with `2379` replaced by
+the correct port number for that server):
+
+```yaml
+apiVersion: projectcalico.org/v3
+kind: GlobalNetworkPolicy
+metadata:
+  name: protected-xyz-server
+spec:
+  selector: protected-xyz-server == "true"
+  order: 500
+  types:
+  - Ingress
+  ingress:
+  - action: Deny
+    protocol: TCP
+    source:
+      notSelector: has(allowed-xyz-client)
```

Comment:
This is a blacklist policy, it assumes that you've got the default allow in place.  I thought this docs was going to cover whitelist policy (or both).

I feel like this rule is going to confuse people, might be worth some explanation.